### ISSUE TITLE
Compile task is up-to-date although error prone version did change

### ISSUE THREAD

**Issue by epeee** [tbroyer/gradle-errorprone-plugin-v0.0.x#72]
Changing the error_prone_core version should end up in compile tasks to be executed.

E.g. let's assume I use
```
    dependencies {
        errorprone 'com.google.errorprone:error_prone_core:2.3.0'
    }
```
and execute `./gradlew build`.

Once I update error_prone_core version to a new version
```
    dependencies {
        errorprone 'com.google.errorprone:error_prone_core:2.3.1'
    }
```
the compile tasks still are up-to-date.

I can confirm that the scenario described does work as expected while using gradle-errorprone-javacplugin-plugin.

**Comment by tbroyer** [comment]
Thanks for the report, I've been suspecting for a while but never took the time to confirm.

Workaround should be
```gradle
tasks.withType(JavaCompile) {
  inputs.files configurations.errorprone
}
```
(at least that's how I'm planning to fix it)

**Comment by tbroyer** [comment]
I released v0.0.15 with the fix.

**Comment by epeee** [comment]
:+1:, thank you!
